.docker/docker_entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Any reason that could be there for using `#!/bin/bash`? Just asking since the entrypoint for ROS DOcker images always seem to use this exact same file.

Reply:
As you can see I've approved the changes, and it doesn't matter for that particular changes. So, that is just a general comment, i.e. for future projects. There is a topic to discuss the difference: https://unix.stackexchange.com/questions/29608/why-is-it-better-to-use-usr-bin-env-name-instead-of-path-to-name-as-my

Reply:
As you can see I've approved the changes, and it doesn't matter for that particular changes. So, that is just a general comment, i.e. for future projects. There is a topic to discuss the difference: https://unix.stackexchange.com/questions/29608/why-is-it-better-to-use-usr-bin-env-name-instead-of-path-to-name-as-my

Reply:
Thanks for the link, I'll have a look!